http.go

```diff
@@ -223,6 +230,21 @@ func NewMonitorClient(accountSid string, authToken string, httpClient *http.Clie
 	return c
 }
 
+func NewTaskRouterClient(accountSid string, authToken string, httpClient *http.Client) *Client {
+	if httpClient == nil {
+		httpClient = &http.Client{Timeout: defaultTimeout}
+	}
+
+	c := newNewClient(accountSid, authToken, TaskRouterBaseUrl, httpClient)
+	c.FullPath = func(pathPart string) string {
+		return "/" + c.APIVersion + "/Workspaces/WS8b69875da105e635f055ab29ac039380/" + pathPart
```

Comment:
`WorkspaceSid` needs to be dynamic, where can we pass this in?

Reply:
I quite like `client.TaskRouter.Workspace("WS123").Activities` but not sure my Go level stretches far enough atm to work it out fully!

Reply:
started work on this but didn't get too far... see cb524ca

Reply:
and eb74d1d

Reply:
I've cherry picked and pushed to this branch (I can rebase once the PR is complete if you wish). Seems the tests are failing because `Client` is `nil` in the `GetResource` function.

Reply:
Apologies for pushing back on you with this PR, learning as I go with Go lang.